upload_form.py

```diff
@@ -186,9 +186,34 @@ def uploaded_file(filename, fileformat=None):
     # column names can't have non-letters in them or javascript fails
     fix_bad_colnames(table)
 
+    column_units=[table[cln].unit for cln in table.colnames]
+    tab_metadata={'Author':'','ADS_ID':'','DOI or URL':'','Submitter':'',
+                  'IsObserved':False,'IsSimulated':False,
+                  'IsGalactic':False,'IsExtragalactic':False}
+    if len(table.meta) > 0:
+        for name, keyword in table.meta['keywords'].items():
+            if name.lower() == 'Author'.lower():
+                tab_metadata['Author']=keyword['value']
+            if (name.lower() == 'ADS_ID'.lower()) or (name.lower() == 'ADS'.lower()):
```

Comment:
these should all be 'elif' to avoid extra executions each time (but this is minor)